components/Feed/ArticleReader/ArticleContent.tsx

```
@@ -0,0 +1,271 @@
+        {shouldUseMarkdown ? (
+          <ReactMarkdown
+            components={components}
+            remarkPlugins={[remarkGfm]}
+            rehypePlugins={[rehypeRaw]}
+            allowedElements={undefined}
+            disallowedElements={[]}
+          >
+            {markdown}
+          </ReactMarkdown>
+        ) : (
+          <div dangerouslySetInnerHTML={{ __html: sanitizedHtml }} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)